### DISCUSSION TITLE
Proof of your academic status

### DISCUSSION THREAD

**Discussion by willwearing** [community/community#23113] [New to GitHub]
<p>Hey team,</p>
<p>I’ve seen that Co-Pilot now requires payments or is free for verified students.</p>
<p>I’m currently studying CS at the Uni of London and just tried to verify my status but can’t take a photo of a letterhead/id as I don’t have a physical one.</p>
<p>Github’s docs seem to be out of date as they show an image upload button which no longer exists: <a href="https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack" class="inline-onebox" rel="noopener nofollow ugc">Apply for a student developer pack - GitHub Docs</a></p>
<p>What are we supposed to do in this case? I don’t have access to a printer and don’t really want to pay 70c for 1 page haha.</p>
<p>Cheers,</p>
<p>Will</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>ask it here</p>
<p><a href="https://github.com/orgs/github-community/discussions/categories/github-education">https://github.com/orgs/github-community/discussions/categories/github-education</a></p>

**Comment by jfullstackdev** [comment]
<p>posting the other similar posts so you can see one another</p>
<aside class="quote" data-post="1" data-topic="257737">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/cant-access-free-github-copilot-from-within-student-account/257737">Can't access free GitHub copilot from within student account</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask questions about your GitHub account and the core features of GitHub.">GitHub Help</span></a>
  </div>
  <blockquote>
    I’m a verified student and am able to access every other student developer pack feature expect GitHub copilot. It shows me subscription fees while trying to access the same. 
Anyone having similar issues?
  </blockquote>
</aside>

<aside class="quote quote-modified" data-post="1" data-topic="257736">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/how-can-students-get-access-to-copilot-through-student-pack/257736">How can students get access to Copilot through Student Pack?</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask questions about your GitHub account and the core features of GitHub.">GitHub Help</span></a>
  </div>
  <blockquote>
    I joined Copilot’s beta half a year ago, but today I found out that it had been commercialized into  a ‘free-trial’ type ordeal. However, it also says that students can have access to copilot for free, through the Student Development pack. I have also joined the Github Global Campus already and I can use the other features of the Student Development pack, but it doesn’t seem to let me use Copilot for free anywhere on the Subscriptions page. Is there any thing I’m missing, or is there anything I …
  </blockquote>
</aside>
